src/Algolia.Search/Serializer/FiltersConverter.cs

```diff
@@ -21,78 +21,90 @@
 * THE SOFTWARE.
 */
 
-using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Text.Json;
+using System.Text.Json.Serialization;
 
 namespace Algolia.Search.Serializer
 {
     /// <summary>
     /// Custom converter for legacy Filters
     /// </summary>
-    public class FiltersConverter : JsonConverter<IEnumerable<IEnumerable<string>>>
+    public class FiltersConverter : JsonConverter<List<List<string>>>
     {
         /// <summary>
-        /// No need to implement this method as we don't want to override default writer
+        /// Using the default writer because only serilization needs custom implementation.
```

Comment:
In that case, couldn't we just get rid of the `Write()` method override altogether?

Reply:
We need to explicitly override it, otherwise the compiler will complain.